cmd/init.go

```diff
@@ -116,6 +116,19 @@ func (o *initOptions) run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// This is simpler with package strings, but that requires Go >=1.21
```

Comment:
What is the purpose of this code? I don't get it.

Reply:
It's a small quality of life consideration. The command works without the prefix, but if one is added, as in `./bitswan-gitops init -v --domain=https://host.io --certs-dir=~/opt/local` gitops`, the cli reports success, but with the endpoints registered as `https://editor.https://host.io`